deploy/clowdapp.yaml

```diff
@@ -5895,6 +5989,13 @@ parameters:
 - displayName: Provider count for batch polling
   name: POLLING_BATCH_SIZE
   value: "100"
+- displayName: XL Provider count for batch polling
+  name: XL_POLLING_BATCH_SIZE
+  value: "10"
+- description: Minimum number of reports before provider classed as XL
+  displayName: Min report count for large providers
+  name: XL_REPORT_COUNT
+  value: "75"
```

Comment:
Just curious how we determined `75` to be the large customer cut off size here.

Reply:
The boolean flag on the provider will be problematic when it comes to configuring this threshold